tests/Doctrine/Tests/ORM/Functional/SchemaValidatorTest.php

```diff
@@ -18,6 +19,16 @@ static public function dataValidateModelSets()
             if ($modelSet == "customtype") {
                 continue;
             }
+
+            // DDC-3380: Register DBAL type for these modelsets
+            if (substr($modelSet, 0, 4) == 'vct_') {
```

Comment:
Value Conversion Type. I shortened this to prevent overly long table-names like `value_conversion_type_inversed_onetomany_compositeid_foreignkey`.